Rework the end of the Install Sensu guide (#2566)
sensu/sensu-docs: content/sensu-go/5.21/installation/install-sensu.md

-d '{
  "check": {
    "metadata": {
      "name": "check-mysql-status"
    },
    "status": 1,
    "output": "could not connect to mysql"
  }
}' \
http://127.0.0.1:3031/events
{{< /code >}}

This request creates a `warning` event that you can [view in your web UI Events page][32].

To create an `OK` event, change the `status` to `0` and resend.
You can change the `output` value to `connected to mysql` to see a different message for the `OK` event.

## Next steps

Now that you have installed Sensu, you’re ready to build your observability pipelines!
Here are some ideas for next steps.

### Do something new with Sensu

If you're ready to see what Sensu can do, one of these pathways can get you started:

- Manually trigger an event that [sends alerts to your email inbox][12].
- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.

### Deploy Sensu outside your local development environment

> If you're going to deploy Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).

Author: ```suggestion
To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.

Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.

No matter whether you deploy a single backend or a clustered configuration, begin by securing Sensu with transport layer security (TLS).
```

> The next step in setting up TLS is to [generate the certificates you need][2].

Author: ```suggestion

The first step in setting up TLS is to [generate the certificates you need][2].
```

> 
> ### Run a Sensu cluster
> 
> If you plan to set up a cluster, here's our suggested pathway:
> 
> 1. [Generate certificates][2].
> 2. [Secure Sensu][8].
> 3. [Run a Sensu cluster][22].

Author: ```suggestion
Then, follow our [Secure Sensu][8] guide to make Sensu production-ready.

After you've secured Sensu, read [Run a Sensu cluster][22] if you are setting up a clustered configuration.
```

## Next steps

Now that you have installed Sensu, you’re ready to build your observability pipelines!
Here are some ideas for next steps.

### Do something new with Sensu

If you're ready to see what Sensu can do, one of these pathways can get you started:

- Manually trigger an event that [sends alerts to your email inbox][12].
- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.

### Deploy Sensu outside your local development environment

To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.

Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.

> However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.

Reviewer: ```suggestion
However, scaling a single backend to a cluster, or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].  For this reason, we recommend you decide whether your deployment will require clustering as part of your initial planning effort.
```

> However, scaling a single backend to a cluster, or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].  For this reason, we recommend you decide whether your deployment will require clustering as part of your initial planning effort.

Author: ```suggestion
However, scaling a single backend to a cluster or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].
For this reason, we recommend that you decide whether your deployment will require clustering as part of your initial planning effort.
```

- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.

### Deploy Sensu outside your local development environment

To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.

Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.

No matter whether you deploy a single backend or a clustered configuration, begin by securing Sensu with transport layer security (TLS).

> 

Author: ```suggestion
```

  }
}' \
http://127.0.0.1:3031/events
{{< /code >}}

This request creates a `warning` event that you can [view in your web UI Events page][32].

To create an `OK` event, change the `status` to `0` and resend.
You can change the `output` value to `connected to mysql` to see a different message for the `OK` event.

## Next steps

Now that you have installed Sensu, you’re ready to build your observability pipelines!
Here are some ideas for next steps.

### Do something new with Sensu

If you're ready to see what Sensu can do, one of these pathways can get you started:

- Manually trigger an event that [sends alerts to your email inbox][12].
- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.

### Deploy Sensu outside your local development environment

To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].

> A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.

Reviewer: ```suggestion
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared database, provided either by Sensu’s embedded etcd or an external etcd cluster.
```

[29]: https://blog.sensu.io/one-year-of-sensu-go/
[30]: ../../reference/backend#initialization
[31]: ../../guides/deploying/
[32]: http://localhost:3000/
[33]: ../../guides/extract-metrics-with-checks/
[34]: https://account.sensu.io/
[35]: ../../api/health/
[36]: #4-open-the-web-ui
[37]: ../../guides/influx-db-metric-handler/
[38]: ../../sensuctl/create-manage-resources/#export-resources

> [39]: ../../sensuctl/create-manage-resources/#create-resources

Reviewer: ```suggestion
[39]: ../../sensuctl/create-manage-resources/#create-resources
[40]: https://etcd.io/docs/v3.3.13/op-guide/runtime-configuration/
```
